src/App.js

```diff
@@ -1,24 +1,57 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react';
+import useFormat from './resources/date';
+import { format } from "date-fns";
+import {
+  BrowserRouter,
+  Routes,
+  Route
+} from "react-router-dom";
+import Home from './pages/Home';
+import Error from './pages/Error';
+
+const NASA_URL= 'https://api.nasa.gov/planetary/apod';
+const KEY = 'p6pWJpQEPAfcfezyNmh4Ru5Bt9onXU4Fb42QXAWa';
+
+const App = () => {
+  const [data, setData] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  const initialDate = format(new Date(), 'yyyy-MM-dd');
+  const [date, setDate] = useState(initialDate);
+  let selected = useFormat(date);
+  
+  let url = `${NASA_URL}?api_key=${KEY}&date=${selected}`;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        const dataImg = await response.json();
+        return setData(dataImg);
+      } catch {
+        console.log(e =>{ throw new Error(e) });
+      }
+    };
+    fetchData();
+    setIsLoading(false);
+  }, [selected]);
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
```

Comment:
```suggestion
    <BrowserRouter basename="/react-apprenticeship-capstone-2">
```
Adding this will tell react-router that this is the base url for this project. You can do the same to your capstone project 1.

Reply:
done 😄